src/components/Organizations/Card.tsx

```diff
@@ -1,80 +1,75 @@
   electionCount?: number
-}
+} & CardProps
+
+/**
+ * Show organization card information. If not id provided will asume it is already inside an OrganizationProvider
+ * @param id If id provided it will fetch the organization from the API
+ * @param electionCount number of elections the organization has
+ * @param rest chakra CardProps
+ * @constructor
+ */
+export const OrganizationCard = ({ id, ...rest }: IOrganizationCardProps) => {
+  if (!id) return <OrganizationCardSkeleton {...rest} />
 
-const OrganizationCard = ({ id, ...rest }: IOrganizationCardProps) => {
   return (
     <OrganizationProvider id={id}>
-      <LargeOrganizationCard id={id} {...rest} />
+      <OrganizationCardSkeleton id={id} {...rest} />
     </OrganizationProvider>
   )
 }
 
-const LargeOrganizationCard = ({ id, electionCount }: IOrganizationCardProps) => {
+const OrganizationCardSkeleton = ({ electionCount: ec, ...rest }: IOrganizationCardProps) => {
   const { organization, loading } = useOrganization()
   const { t } = useTranslation()
 
-  return (
-    <Card direction={'row'} alignItems='center' overflow={'scroll'} pl={4}>
-      <Box w={'50px'}>
-        <Avatar
-          mx='auto'
-          fallbackSrc={'/images/fallback-account-dark.png'}
-          alt={t('organization.avatar_alt', {
-            name: organization?.account.name.default || organization?.address,
-          }).toString()}
-        />
-      </Box>
-      <CardBody>
-        {loading ? (
-          <Text fontWeight={'bold'} wordBreak='break-all' size='sm'>
-            {id}
-          </Text>
-        ) : (
-          <OrganizationName fontWeight={'bold'} wordBreak='break-all' size='sm' />
-        )}
-        <ReducedTextAndCopy color={'textAccent1'} toCopy={id}>
-          {id}
-        </ReducedTextAndCopy>
-        <Text fontSize={'sm'}>
-          <Trans i18nKey={'organization.process_count'} count={electionCount}>
-            <strong>Process:</strong> {{ count: electionCount }}
-          </Trans>
-        </Text>
-      </CardBody>
-    </Card>
-  )
-}
-
-export const SmallOrganizationCard = ({ id }: IOrganizationCardProps) => {
-  const { organization } = useOrganization()
-  const { t } = useTranslation()
+  const electionCount = ec ?? organization?.electionIndex
+  const pid = organization?.address
 
-  const name = organization?.account.name.default || organization?.address
+  if (!pid) return null
 
   return (
-    <Flex direction={'row'} alignItems='center' overflow={'scroll'} gap={2}>
-      <Box w={'25px'}>
-        <Avatar
-          mx='auto'
-          fallbackSrc={'/images/fallback-account-dark.png'}
-          alt={t('organization.avatar_alt', {
-            name: organization?.account.name.default || organization?.address,
-          }).toString()}
-        />
-      </Box>
-      <ReducedTextAndCopy color={'textAccent1'} size='sm' toCopy={id}>
-        {name}
-      </ReducedTextAndCopy>
-    </Flex>
+    <Link href={generatePath(organizationPath, { pid })}>
```

Comment:
This should have a `as={RouterLink}` and use `to` instead of href. Href is only for external links, everything else should be linked using the react-router-dom link and the `to` property.

Reply:
fbf94a5